ossm/merge_upstream.sh

```diff
@@ -45,7 +45,7 @@ merge() {
 
   set +e # git returns a non-zero exit code on merge failure, which fails the script
   if [ "${MERGE_STRATEGY}" == "merge" ]; then
-    git -c "user.name=$GIT_USERNAME" -c "user.email=$GIT_EMAIL" merge --no-ff -m "$GIT_COMMIT_MESSAGE" --log upstream/"$MERGE_BRANCH"
+    git -c "user.name=$GIT_USERNAME" -c "user.email=$GIT_EMAIL" merge --no-ff -m "$GIT_COMMIT_MESSAGE" --log upstream/"$MERGE_BRANCH" --strategy-option theirs
```

Comment:
I don't think we want theirs here. We are resolving the conflicts by running 'make gen' when the merge fails. With 'theirs', the merge will never fail.

Reply:
the problem is that there are conflicts in `api/v1alpha/**types.go` files. If they're not resolved, `make gen` fails. We could change the script to always run `make gen` after merging.

Reply:
If we don't mind one extra commit after "Automated merge" commit, it's fine. I don't see any other way right now.